changelogs/fragments/2220_nmcli_wifi_support.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- "nmcli - add ability to connect to a Wifi network and also to attach the connection to a master (bond) (https://github.com/ansible-collections/community.general/pull/2220)."
```

This change makes it more readable and easier to understand IMHO, but it's fine as it is now as well.
```suggestion
- "nmcli - add ability to connect to a Wifi network and attach it to a master (bond) (https://github.com/ansible-collections/community.general/pull/2220)."
```

---

Indeed. Thanks.